README.md

```diff
@@ -704,6 +704,15 @@ params do
   optional :non_random_number, type: Integer, default:  Random.rand(1..100)
 end
 ```
+Default value provided by lambda can take zero, one or two arguments
+
+```ruby
+params do
+  optional :random_number, type: Integer, default: -> () { Random.rand(1..100) }
+  optional :random_number, type: Integer, default: -> (request) { Random.rand(1..100) }
+  optional :random_number, type: Integer, default: -> (request, cookies) { Random.rand(1..100) }
```

Comment:
I think the proc should only take `request`, from which you should be able to get to cookies and what not. Request is basically everything, and if it's not we should be passing in context or a hash of options.